index.html

```diff
@@ -8,75 +8,89 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <header class="portfolio">
+  <header class="portfolio">
 		<a href="#">S</a>
-			<nav class="nav-menu">
-				<a href="#"><img src="Assets/Icon-Menu.png" alt=""></a>
+			<nav class="nav-list">
+        <ul class="nav-menu">
+          <li><a href="#recent">Portfolio</a></li>
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#form-footer">Contact</a></li>
+          <li><a href="#"><img src="Assets/Letter.png" alt=""></a></li>
+      </ul>
+				<a class="icon-menu" href="#"><img src="Assets/Icon-Menu.png" alt=""></a>
 			</nav>
-	</header>
-
-		<div class="personal-info">
+	</header> 
+  
+  <section class="personal-info">
+    <div class="brief-intro">
 			<h1>Hey There. <br> I am Shakur</h1>
 			<h3>I am a Software Developer</h3>
 			<p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
 				don’t hestiate to contact me.</p>
+      </div>
 				<ul class="links">
 					<li class="lists"><a href="#"> <img src="Assets/github.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Linkedin.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Vector.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/twitter.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/medium.png" alt=""></a></li>
 				</ul>

```

Comment:
Kindly increase the size of the image on the left of the section as shown in the design below 

#### Original design
![right1](https://user-images.githubusercontent.com/58376020/122623323-e545d780-d09b-11eb-9105-3b116fd07529.png)

#### Your Project
![wrong1](https://user-images.githubusercontent.com/58376020/122623335-f1319980-d09b-11eb-808c-50e2cdc643fb.png)

Reply:
fixed 👍